### DISCUSSION TITLE
Add Unicode Support to the Editor

### DISCUSSION THREAD

**Discussion by Denneisk** [wiremod/wire#2755] [Suggestions]
Yes, #2237 but without it getting reverted.

**Comment by vurvdev** [comment]
Would be easier to invest in a new editor entirely tbh, although I'm not exactly keen on my editor supporting unicode, unless I was able to find a way for it not to screw up everything and make the codebase significantly more complex.

https://github.com/Vurv78/veditor

**Reply by thegrb93** [reply]
Are the html based editors any better lately? Embedded vscode would be cool.

**Reply by vurvdev** [reply]
Nothings wrong with the editors themselves, just that ~~facepunch~~ rubat hasn't and likely never will make chromium the default branch.

And the vast majority of gmod users are on the default branch

**Reply by thegrb93** [reply]
Yea I think most serious coders just use an external editor.
